conversion/conversion.go

```diff
@@ -213,10 +214,22 @@ func dataFromSQL(driver string, config spanner.BatchWriterConfig, client *sp.Cli
 	return writer, nil
 }
 
+func getDynamoDBClientConfig() aws.Config {
+	cfg := aws.Config{}
+	endpointOverride := os.Getenv("DYNAMODB_ENDPOINT_OVERRIDE")
+	if endpointOverride != "" {
+		cfg = aws.Config{
+			Endpoint: aws.String(endpointOverride),
+		}
+	}
+	return cfg
+}
+
 func schemaFromDynamoDB(sampleSize int64) (*internal.Conv, error) {
 	conv := internal.MakeConv()
 	mySession := session.Must(session.NewSession())
-	client := dydb.New(mySession)
+	cfg := getDynamoDBClientConfig()
+	client := dydb.New(mySession, &cfg)
```

Comment:
consider inlining getDynamoDBClientConfig() since cfg isn't used again later in the code; similarly in dataFromDynamoDB below